target/linux/generic/files/drivers/mtd/mtdsplit/mtdsplit_uimage.c

```diff
@@ -55,6 +64,25 @@ struct uimage_header {
 	uint8_t		ih_name[IH_NMLEN];	/* Image Name		*/
 };
 
+/*
+ * Asus-specific header add-on
+ * On some devices such as RP-AC56, Asus stores an additional data
+ * in ih_name field, in paricular, an offset to rootfs can be
+ * found there
+ */
+typedef struct {
+        uint8_t major;
+        uint8_t minor;
+} version_t;
+
+typedef struct {
+        version_t       kernel;
+        version_t       fs;
+        uint8_t         productid[IH_PRODLEN];
+        uint8_t         sub_fs;
+        uint32_t        ih_ksz;
+} asus_t;
```

Comment:
Use tab indent as for the other struct/typedef

Reply:
done